SparkPost/2web2ui
# SA-1588 - Typeahead includes custom reports

### What Changed
 - Action/Reducer for API request added
 - Custom autocomplete select added

### How To Test
 - Go to Report Builder
 - On top of report builder, select preset reports
 - Blur and focus typeahead again
   - Field should empty and allow for new typeahead

### To Do
- [x] Make sure list items are ordered correctly
- [x] Responsiveness of list items
- [x] Fix test
